tutorial/source/svi_part_iii.ipynb

```diff
@@ -108,7 +108,7 @@
     "svi = SVI(model, guide, optimizer, TraceGraph_ELBO())\n",
```

Comment:
<ul><li>"A equals B" or "A is equal to B" but not "A equals to B"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/pyro/pull/3197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Fixed it.